endtoendtests/pom.xml

```diff
@@ -13,15 +13,14 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.source>1.8</maven.compiler.source>
-		<maven.compiler.target>1.8</maven.compiler.target>
-		<azure.functions.maven.plugin.version>1.17.0</azure.functions.maven.plugin.version>
+		<java.version>1.8</java.version>
+		<azure.functions.maven.plugin.version>1.18.0</azure.functions.maven.plugin.version>
 		<azure.functions.java.library.version>2.0.1-SNAPSHOT</azure.functions.java.library.version>
 		<functionAppName>azure-functions-java-endtoendtests</functionAppName>
 		<functionAppRegion>westus</functionAppRegion>
 		<functionResourceGroup>java-functions-group</functionResourceGroup>
-		<stagingDirectory>${project.build.directory}/azure-functions/${functionAppName}</stagingDirectory>
-		<allowTelemetry>false</allowTelemetry>
+<!--		<stagingDirectory>${project.build.directory}/azure-functions/${functionAppName}</stagingDirectory>-->
```

Comment:
We can remove this as well if not necessary

Reply:
sorry missed this one, will remove them in next pr